modules/_node-scripts/bin.js

```diff
@@ -9,13 +9,25 @@
 const fs = require('fs');
 const path = require('path');
 
-const COMMAND_DESCRIPTIONS = {
-	build: 'builds frontend stuff of current project',
+const COMMANDS = {
+	'build': {
+		description: 'builds frontend stuff of current project',
+		parameters: '',
+	},
+	'check:tsc': {
+		description:
+			'runs TypeScript checks in the current project or globally (if run from modules)',
+		parameters: '[<tsc arguments>]',
+	},
+	'generate:tsconfig': {
```

Comment:
Should we be running "generate" whenever we run "check:tsc" as well?

Reply:
see https://github.com/liferay-frontend/liferay-portal/pull/4229#discussion_r1620224680